aem-authoring-toolkit-plugin/src/main/java/com/exadel/aem/toolkit/core/handlers/container/TabsHandler.java

```diff
@@ -61,29 +63,22 @@ public void accept(Class<?> componentClass, Element parentElement) {
             return;
         }
         List<Field> allFields = PluginReflectionUtility.getAllNonStaticFields(componentClass);
-        Element firstTabElement = null;
         for (int i = 0; i < dialogTabs.length; i++) {
             Tab tab = dialogTabs[i];
             String nodeName = getXmlUtil().getUniqueName(tab.title(), DEFAULT_TAB_NAME, tabItems);
             Element tabElement = (Element) tabItems.appendChild(getXmlUtil().createNodeElement(nodeName));
             tabElement.setAttribute(JcrConstants.PN_TITLE, tab.title());
             appendAttributes(tabElement, tab);
-            // the first tab will contain the elements explicitly assigned to it
-            // and also all the elements with no tab or erroneous tab specified
-            // so we render the first tab after all the rest
-            if (i == 0) {
-                firstTabElement = tabElement;
-                continue;
-            }
+            int finalI = i;
```

Comment:
Not pretty, you know... I would rather declared here `final boolean isFirstTab = i == 0` and then passed the boolean